### PR TITLE
Navigation: Open menu in the desktop way

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

Adds the required CSS to open to new header menu as a mega menu, instead of a sidebar. It creates a mess for the items on desktop (see screenshot), but allows us, to split the work.

None of the other screen widths should see any visible change.

PS: These commit messages are dumb - will squash this before merge. 😖 

## What is the value of this and can you measure success?

Small steps towards a beautiful desktop header.

## Does this affect other platforms - Amp, Apps, etc?

No.

## Screenshots

**Closed**

screen shot 2017-06-05 at 13 47 59

**Open**

screen shot 2017-06-05 at 14 24 41

## Tested in CODE?

No.

### CODE REVIEW

File: static/src/stylesheets/layout/new-header/_menu.scss [hunk 1]
```diff
@@ -35,8 +38,27 @@
         margin-right: $gutter-xlarge + ($veggie-burger-medium / 2);
     }
 
+    // based on https://css-tricks.com/full-width-containers-limited-width-parents/#article-header-id-6
+    @include mq(desktop) {
+        display: none;
+        left: 50%;
+        margin-left: -50vw;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu.scss]: We only have [partial support](https://caniuse.com/#search=Viewport%20units) for vw on edge and ie. Have you taken a look to see what happens when this isn't supported?

File: static/src/stylesheets/layout/new-header/_menu.scss [hunk 2]
```diff
@@ -35,8 +38,27 @@
         margin-right: $gutter-xlarge + ($veggie-burger-medium / 2);
     }
 
+    // based on https://css-tricks.com/full-width-containers-limited-width-parents/#article-header-id-6
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu.scss]: That is a really cool trick!

File: static/src/stylesheets/layout/new-header/_menu.scss [hunk 3]
```diff
@@ -35,8 +38,27 @@
...
+        width: 100vw;
+    }
+
     .new-header--open & {
-        @include menu-animation(0%);
+        @include mq($until: desktop) {
+            @include menu-animation(0%);
+        }
+
+        // TODO: replace with sexy animation
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu.scss]: 💯

**Review by NataliaLKB — APPROVED** [review]
Awesome step in the right direction 👍

File: static/src/stylesheets/layout/new-header/_menu.scss (hunk 1 above)
Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_menu.scss]: Haven't done any cross-browser testing so far. I'd say that we should start doing this, once we have the rough layout in place, so we can develop fast now and towards the end start fixing bugs ...

Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu.scss]: Fine by me, but lets keep it in mind as we go.

**Pull request merged**